cmake/std/atdm/apps/empire/EMPIRETrilinosEnables.cmake

```diff
@@ -10,6 +10,9 @@ ATDM_SET_CACHE(Gtest_SKIP_INSTALL TRUE CACHE BOOL)
 # #5341.  (Once we can get SPARC to have its own gtest, then we can move this
 # setting to ATDMDevEnvSettings.cmake. See SPAR-614.)
 
+ATDM_SET_CACHE(Trilinos_ENABLE_Gtest FALSE CACHE BOOL)
+ATDM_SET_CACHE(Kokkos_ENABLE_Profiling TRUE CACHE BOOL)
```

Comment:
FYI: You don't need to set this since profiling is on by default in Trilinos.  You can see this is set in the ATDM Trilinos builds already by looking [here](https://testing.sandia.gov/cdash/viewNotes.php?buildid=5888339#!#note3) for example showing:

```
Kokkos_ENABLE_Profiling:BOOL=ON
```

It is fine to leave it in, but it is not necessary.